opm/autodiff/BlackoilModelEbos.hpp

```diff
@@ -1206,7 +1194,130 @@ namespace Opm {
             return values;
         }
 
-        const FIPData& getFIPData() const {
+        SimulationDataContainer getSimulatorData () const
+        {
+            typedef std::vector<double> VectorType;
+
+            const auto& ebosModel = ebosSimulator().model();
+            const auto& phaseUsage = fluid_.phaseUsage();
+
+            // extract everything which can possibly be written to disk
+            const int numCells  = ebosModel.numGridDof();
+            const int numPhazes = numPhases();
```

Comment:
I'll admit that makes me uneasy (and I guess that is one of the reasons I prefer to name variables like `num_phases`), perhaps it is better to just use the function calls in there?